js/grid/generate_grid.js

```diff
@@ -121,6 +131,7 @@ function appendTableData(trFall, trSpring, time) {
             .attr("satisfied", "true")
             .addClass("timetable-cell"));
     }
-
+    
```

Comment:
This is an indication you have added some trailing whitespace; this should be removed.